### DISCUSSION TITLE
Interacting with selected text and highlighter

### DISCUSSION THREAD

**Discussion by Gegsl** [xournalpp/xournalpp#5477] [Q&A] [answered]
Hey everybody,

I have a question considering the tool "interacting with selected text" and the highlighter. Is it possible to highlight the text in the pdf directly without first using the "Select Linear Text Tool". It would be really nice to skip that step and just pick the highlighter. 

Thank you very much in advance

**Comment by rolandlo** [comment, marked as answer]
Currently it is not possible. I agree though that it would be handy to be able to select the text without switching tools first.
